compiler/astalgo.nim

```diff
@@ -353,6 +353,12 @@ proc symToYaml(conf: ConfigRef; n: PSym, indent: int = 0, maxRecDepth: int = - 1
 
 import tables
 
+const backrefStyle = "\e[90m"
+const enumStyle = "\e[34m"
```

Comment:
Using `terminal`, you can do `ansiForegroundColorCode(fgBlue)` instead of typing `"\e[34m" `. That is much more readable as it is evident that you want to style the enums in blue color.